Source/Parser/JsonLDParser.swift

```diff
@@ -57,32 +57,29 @@ class JsonLDParser {
 	}
 	
 	static func flatten(
-		context: Context, foldedAttributes: inout Dict, attributes: Attributes) -> Dict {
+		context: Context, foldedAttributes: inout Dict, attribute: Attribute) -> Dict {
 		
-		let key = attributes.keys.first ?? ""
-		let value = attributes.values.first ?? ""
-		
-		var attributes = foldedAttributes["attributes"] as? Dict ?? Dict() 
+        var attributes = foldedAttributes["attributes"] as? Dict ?? Dict()
 		var things = foldedAttributes["things"] as? Dict ?? Dict()
+        
+        let key = attribute.keys.first ?? ""
+        let value = attribute.values.first ?? ""
 		
-		if let value = value as? Dict {
-			parseObject(key: key, value: value, context: context, attributes: &attributes, 
-						things: &things)
+        if let value = value as? [String : Any?] {
+			parseObject(key: key, value: value, context: context, attributes: &attributes, things: &things)
 		}
-		else if let value = value as? [[String: Any]] {
-			parseObjectArray(key: key, value: value, context: context, attributes: &attributes, 
-							 things: &things)
+        else if let value = value as? [[String : Any]] {
+			parseObjectArray(key: key, value: value, context: context, attributes: &attributes, things: &things)
 		}
-		else if isId(name: key, context: context) {
+        else if isId(name: key, context: context) {
 			let value =  value as? String ?? ""
 			let relation = Relation(id: value, thing: nil)
 			
-			let null: Any? = nil
-			things[value] = null
+			things[value] = nil
 			
 			attributes[key] = relation
 		}
-		else {
+        else {
```

Comment:
While we don't have a defined SF, it may be easier to keep as it is for now. Can you revert this hunk?